site/en/blog/storage-partitioning-deprecation-trial/index.md

```diff
@@ -0,0 +1,151 @@
+
+Below is an overview of the deprecation trial and what to expect. If you have
+feedback to share or you experience any issues throughout this trial let us know
+in the
+[Partitioned Storage Deprecation Trial Github repository](https://github.com/miketaylr/partitioned-storage-deprecation-trial-feedback).
+
+### DisableThirdPartyStoragePartitioning
+
+The following APIs will remain unpartitioned in third-party contexts should you
+enroll the top-level site in the `DisableThirdPartyStoragePartitioning`
+deprecation trial:
+[Storage APIs](https://github.com/wanderview/quota-storage-partitioning/blob/main/explainer.md#storage-apis)
+(such as localStorage, sessionStorage, IndexedDB, Quota, and other),
+[Communication APIs](https://github.com/wanderview/quota-storage-partitioning/blob/main/explainer.md#communication-apis)
+(such as BroadcastChannel, SharedWorkers, and WebLocks), and
+[ServiceWorker API](https://github.com/wanderview/quota-storage-partitioning/blob/main/explainer.md#serviceworker-api).
+
+{% Aside 'caution' %}
+For this `DisableThirdPartyStoragePartitioning` trial, the origin trial token must be included via an HTML `<meta>` tag and not an `Origin-Trial` HTTP header.
+{% endAside %}
+
+Example:
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/dqBSsNU8W3HOmMkPDYYh.png", alt="Storage partitioning diagram", width="800", height="544" %}
+
+For a more detailed explanation, check out the
+[project explainer](https://github.com/wanderview/quota-storage-partitioning/blob/main/explainer.md).
+
+### DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning
+
+If you enroll in the
+`DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning`
+deprecation trial, navigating a tab to an enrolled origin will cause all
+cross-site iframes of that same origin to remain unpartitioned only for
+`Window.sessionStorage` only and only for the lifetime of that particular tab.
+Whereas the `DisableThirdPartyStoragePartitioning` deprecation trial affects all
+third-party contexts embedded within the enrolled origin, the
+`DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning`
+deprecation trial instead registers a given origin to receive unpartitioned
+access when embedded in third-party contexts.
+
+{% Aside %}
+The `DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning` trial will work via HTML `<meta>` or an `Origin-Trial` HTTP header.
+{% endAside %}
+
+Example:
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/6uxCgy48dDcfFVgkwycu.png", alt="Storage partitioning diagram after general paritoning.", width="800", height="432" %}
+
+## What does this mean for web developers?
+
+Sites should audit their usage of unpartitioned storage, service worker, and
+communication APIs in third-party contexts, and, if needed, prepare for
+third-party partitioning before these deprecation trials expire. **The intent is
+to expire these deprecation trials with M123 on March 5, 2024.**
```

Comment:
Now M124 on April 2, 2024

Reply:
Hey sorry - I was wrong - the OT registration site states: M123 ending on May 2, 2024. https://developer.chrome.com/origintrials/#/view_trial/-8517432795264450559